source/agora/common/Config.d

```diff
@@ -620,7 +620,7 @@ private ValidatorConfig parseValidatorConfig (Node* node, in CommandLine cmdln)
     const preimage_reveal_interval = get!(Duration, "validator", "preimage_reveal_interval",
         str => str.to!ulong.seconds)(cmdln, node);
     const nomination_interval = get!(Duration, "validator", "nomination_interval",
-        str => str.to!ulong.msecs)(cmdln, node);
+        str => str.to!ulong.secs)(cmdln, node);
```

Comment:
Should be `seconds`.

Reply:
Ahh I was wondering why the compiler doesn't like it..